docs/code-examples/text_document.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+"""Log a simple set of line segments."""
+
+import rerun as rr
+import rerun.experimental as rr2
+
+rr.init("rerun_example_text_document", spawn=True)
+
+rr2.log("text_document", rr2.TextDocument(
+    body="Hello, TextDocument!",
```

Comment:
remove the comma so `black` formats it all on a single line
```suggestion
    body="Hello, TextDocument!"